include/swift/AST/AccessScope.h

```diff
@@ -0,0 +1,89 @@
+//===--- AccessScope.h - Swift Access Scope ---------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the AccessScope class.
```

Comment:
This doesn't really add anything beyond the file header (which is also pretty sparse). If there's nothing more useful to say here, you can just remove it.